diff --git a/chrome/browser/net/stub_resolver_config_reader.cc b/chrome/browser/net/stub_resolver_config_reader.cc
index 14db5800fac2adbd6566802c34047283e00e956d..40a3e9dcd89791f6ec303cfffa7ae395e037fc44 100644
--- a/chrome/browser/net/stub_resolver_config_reader.cc
+++ b/chrome/browser/net/stub_resolver_config_reader.cc
@@ -131,6 +131,8 @@ StubResolverConfigReader::StubResolverConfigReader(PrefService* local_state,
         default_secure_dns_mode = net::SecureDnsMode::kSecure;
       }
       default_doh_templates = features::kDnsOverHttpsTemplatesParam.Get();
+      
+      BRAVE_GET_AND_UPDATE_CONFIGURATION
     }
     local_state_->SetDefaultPrefValue(
         prefs::kDnsOverHttpsMode,
